code/index.html

```diff
@@ -4,15 +4,53 @@
     <meta charset="utf-8" />
     <title>Project Name</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
   <body>
     <header>
-      <!-- If you want a header with a title, logo and a navbar, here is where you should add that. -->
-    </header>
+      
+      <h1>Amy Brown: Medieval Literatures, Modern Cultures</h1>
+      <img src="header-image-crop.jpg" class="headerimage" alt="a red cushioned chair holding books on Old and Middle English literature and medieval sexuality">
+    
+      </header>
 
-    <main class="">
-      <!-- The main container to use as the flex parent for your card layout, the main content of the site should be in here.  -->
+    <main class="parent">
+      <div class="researchcard">
+        <h2>Current Research</h2>
+        <p><ul><li>Contemporary Chaucer, adaptation and reception, and Chacuerian obscentiy in context.</li>
+        <li>Friendship between men and women in medieval romance; adaptation between French and English in the middle ages.</li>
+        <li>Gender and violence in premodern literature and modern adaptations.</li>
+      </ul>
+      <a href=DUMMYLINK>Publications</a> | <a href=DUMMYLINK>Conference Papers</a>
+    </p>
```

Comment:
The list should not be wrapped in a paragraph, and remember the indentation. Take a look at our [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) to read more.
```
<ul>
  <li>Contemporary Chaucer, adaptation and reception, and Chacuerian obscentiy in context.</li>
  <li>Friendship between men and women in medieval romance; adaptation between French and English in the middle ages.</li>
  <li>Gender and violence in premodern literature and modern adaptations.</li>
</ul>
<p>
  <a href=#>Publications</a>
  |
  <a href=#>Conference Papers</a>
</p>
```